src/api/app/models/backend_package.rb

```diff
@@ -20,7 +20,7 @@ class BackendPackage < ApplicationRecord
   # this is called from the UpdatePackageMetaJob and clockwork
   def self.refresh_dirty
     Package.dirty_backend_package.pluck(:project_id).uniq.each do |project_id|
-      Project.find(project_id).delay(priority: 10).update_packages_if_dirty
+      Project::UpdatePackagesIfDirtyJob.perform_later(project_id)
```

Comment:
`Project.find(project_id).update_packages_if_dirty.perform_later`

That doesn't work. I dont think active job has any equivalent to the `.delay` method that delayed jobs has. AFAIK every async job needs to be in a job class.